.ui/library/lviPerson.cpx

```diff
@@ -25,15 +23,63 @@
 			}
 		},
 		{
-			"className": ".sf-label .sf-listViewItem-simple-title",
+			"className": ".sf-flexLayout .sf-listViewItem-simple .sf-listViewItem-inactive .sf-listViewItem-active",
```

Comment:
Both inactive and active classes are here. You might want to remove one of them.

Reply:
I think we don't need both! Because listView will render what need. I will remove both.